packages/front-end/pages/settings/customfields.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { useEffect, useMemo, useState } from "react";
+import { CustomField } from "back-end/types/organization";
+import {
+  arrayMove,
+  SortableContext,
+  verticalListSortingStrategy,
+} from "@dnd-kit/sortable";
+import {
+  closestCenter,
+  DndContext,
+  DragOverlay,
+  KeyboardSensor,
+  MouseSensor,
+  TouchSensor,
+  useSensor,
+  useSensors,
+} from "@dnd-kit/core";
+import { useUser } from "@/services/UserContext";
+import { useCustomFields } from "@/services/experiments";
+import { useAuth } from "@/services/auth";
+import track from "@/services/track";
+import {
+  SortableCustomFieldRow,
+  StaticCustomFieldRow,
+} from "@/components/Experiment/SortableCustomField";
+import usePermissions from "../../hooks/usePermissions";
```

Comment:
nit: these sorts of paths can be "@/hooks/usePermissions". I know WebStorm doesn't automatically catch these. 😢